packages/graphin-components/src/CreateEdge/index.tsx

```diff
@@ -47,9 +47,11 @@ const CreateEdge: React.FunctionComponent<Props> = props => {
 
     graph.on('aftercreateedge', handleAftercreateedge);
     return () => {
-      graph.removeBehaviors('create-edge', 'default');
-      graph.get('canvas').setCursor('default');
-      graph.off('aftercreateedge', handleAftercreateedge);
+      if (graph && !graph.destroyed) {
```

Comment:
Return early because then you do not have to nest code with brackets that makes code harder to read.

```js
if (!graph || graph.destroyed) return;

graph.removeBehaviors('create-edge', 'default');
graph.get('canvas').setCursor('default');
graph.off('aftercreateedge', handleAftercreateedge);
```